### DISCUSSION TITLE
建议将演示demo做一个仓库

### DISCUSSION THREAD

**Discussion by lxglife** [xuexiangjys/XUpdate#130] [Ideas]
在XUpdate的代码中有相关的使用demo,但是没有演示demo的齐全，建议将演示demo的代码建立一个仓库，并在配套设置中进行超链接。

**Comment by xuexiangjys** [comment]
XUpdateAPI里面有详细的使用步骤:
https://github.com/xuexiangjys/XUpdateAPI
